server/src/test/java/org/opensearch/search/SearchIpFieldTermsTests.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.search;
+
+import org.opensearch.action.bulk.BulkRequestBuilder;
+import org.opensearch.action.search.SearchResponse;
+import org.opensearch.common.network.InetAddresses;
+import org.opensearch.common.xcontent.XContentFactory;
+import org.opensearch.core.xcontent.XContentBuilder;
+import org.opensearch.index.query.QueryBuilders;
+import org.opensearch.test.OpenSearchSingleNodeTestCase;
+import org.hamcrest.MatcherAssert;
+
+import java.io.IOException;
+import java.net.InetAddress;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+import static org.opensearch.action.support.WriteRequest.RefreshPolicy.IMMEDIATE;
+import static org.hamcrest.Matchers.equalTo;
+
+public class SearchIpFieldTermsTests extends OpenSearchSingleNodeTestCase {
```

Comment:
It might be too heavy testing approach for the current change. However, such depth is necessary for the second stage (when ranges will be combined at PointsVisitor level). 
There might be just a `IpFieldTypeTest` asserting resulting BooleanQuery has set of IP points. WDYT?